api/envoy/extensions/filters/http/router/v3/router.proto

```diff
@@ -21,11 +21,21 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // Router :ref:`configuration overview <config_http_filters_router>`.
 // [#extension: envoy.filters.http.router]
 
-// [#next-free-field: 9]
+// [#next-free-field: 10]
 message Router {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.config.filter.http.router.v2.Router";
 
+  message UpstreamAccessLogOptions {
+    // If set to true, an upstream access log will be recorded when a new HTTP request is received,
```

Comment:
this could be misread as 2 additional logs, one on new stream and one after connection
maybe rephrase "will be recorded when an upstream stream is associated to an http request"?